site/content/docs/main/style-guide.md

```diff
@@ -10,55 +10,58 @@ This page outlines writing style guidelines for the Velero documentation and you
 The Velero documentation uses the [kramdown](https://kramdown.gettalong.org/) Markdown renderer.
 
 ## Content best practices
-
-
 ### Use present tense
 
-<table caption="Do and Don't - Use present tense">
-  <tr><th>Do</th><th>Don't</th></tr>
-  <tr><td>This `command` starts a proxy.</td><td>This command will start a proxy.</td></tr>
-</table>
+{{< table caption="Do and Don't - Use present tense" >}}
+|Do|Don't|
+|--- |--- |
+|This `command` starts a proxy.|This command will start a proxy.|
+{{< /table >}}
 
 Exception: Use future or past tense if it is required to convey the correct meaning.
 
 ### Use active voice
 
-<table caption="Do and Don't - Use active voice" >
-  <tr><th>Do</th><th>Don't</th></tr>
-  <tr><td>You can explore the API using a browser.</td><td>The API can be explored using a browser.</td></tr>
-  <tr><td>The YAML file specifies the replica count.</td><td>The replica count is specified in the YAML file.</td></tr>
-</table>
+{{< table caption="Do and Don't - Use active voice" >}}
```

Comment:
These don't seem to render well outside of the website.

Reply:
also, this page may not be reachable from the website.

Reply:
i'm not sure what you mean here, can you provide more context?

Reply:
https://github.com/vmware-tanzu/velero/blob/ed40bf378a4479d8d6ba14194c892676af54b8e8/site/content/docs/main/style-guide.md
This is a markdown render of this file and the `{{< table caption="Do and Don't - ...` shortcodes are not rendered correctly.
Looks like these shortcodes are useful for markdown rendered by hugo, which is the website only for Velero.

To answer my question. Yes, this page is reachable on the website at https://velero.io/docs/main/style-guide

Reply:
I don't think they're meant to render outside of the website. That they did on github otherwise was a nice side effect.

If we introduce the requirement that everything must render outside of Hugo, then we can't use shortcodes and other Hugo features.

Reply:
That's reasonable.